Cathay/Features/AllMovies/Views/SwiftUIs/AllMoviesView.swift

```diff
@@ -31,3 +31,7 @@
         AllMoviesView(movies: MovieModel.mockData).previewDevice(PreviewDevice.init(rawValue: "iPhone SE"))
     }
 }
+
+
+
+
```

Comment:
<!--
  0 failure: 
  2 warning:  Files should have..., Limit vertical wh...
  
  
  DangerID: danger-id-Danger;
  File: Cathay/Features/AllMovies/Views/SwiftUIs/AllMoviesView.swift;
  Line: 37;
-->

- :warning: Files should have a single trailing newline. (`trailing_newline`)
- :warning: Limit vertical whitespace to a single empty line. Currently 4. (`vertical_whitespace`)